runtime/doc/develop.txt

```diff
@@ -760,4 +765,8 @@ features can be disabled at compile time through an appropriate switch.  After
 a new release, deprecated features may be removed completely in a following
 cycle.
 
+Some patch in development cycle maybe found it was wrong or had outage, then
+it should be backout, to reduce some mess happening, it should be reverted in
+maximally one week.
```

Comment:
How about this:
> If a patch introduced during development is found to be wrong or causes issues, it should be reverted to avoid creating further problems.

I'll skip the time frame because:
- it doesn't work well with me being on vacation
- not always will such issues be identified within a week and as such this is a bad idea to spell it out explicitly. 
  It may happen after a longer time only and we shouldn't restrict ourselves to this time frame. But it should be done within the actual development cycle.

Reply:
the "time frame" actually was not to you, but to PR requester, if you noticed there were some problematic (merged) PR were existed long time but didnot reverted (tho had been reverted done recently), so it was a *wish* to limit the PR requester to backout bad PR asap.

Reply:
i think you do :smile: for example `https://github.com/vim/vim/pull/18213`, whatever `preinsert` or `ComplMatchIns`, one of them was incorrect naming, but now it is in mid-stage, neither of them was changing. :shrug: